shared/src/iosMain/kotlin/com/intive/picover/shared/MainViewController.kt

```diff
@@ -1,7 +1,31 @@
 package com.intive.picover.shared
 
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Text
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
 import androidx.compose.ui.window.ComposeUIViewController
 import com.intive.picover.common.loader.PicoverLoader
+import dev.icerock.moko.resources.compose.stringResource
 
-@Suppress("ktlint:standard:function-naming")
-fun MainViewController() = ComposeUIViewController { PicoverLoader() }
+object MainViewController {
+
+	fun provide() = ComposeUIViewController {
+		Column(
+			modifier = Modifier
+				.fillMaxSize()
+				.background(Color.LightGray),
+			horizontalAlignment = Alignment.CenterHorizontally,
+			verticalArrangement = Arrangement.spacedBy(10.dp),
+		) {
+			Text(text = "Compose Multiplatform Sandbox")
+			PicoverLoader()
+			Text(text = stringResource(MR.strings.GenericErrorMessage))
```

Comment:
Another lesson is to try use shared code as soon as it's configured to check if setup works as expected 😄